docs/jwst/associations/jwst_conventions.rst

```diff
@@ -52,14 +52,15 @@ Level 3 pipeline module to use to process this association.
 
 The current association types are:
 
+  * `image2`: Intended for `calwebb_image2` processing
   * `image3`: Intended for `calwebb_image3` processing
-  * `spec3`: Intended for `calwebb_spec3` processing
-  * `wfs`: Wave front sensing data, used by `wfs_combine`
   * `ami3`: Intended for `calwebb_ami3` processing
   * `coron3`: Intended for `calwebb_coron3` processing
+  * `tso-image2`: Intended for `calwebb_tso_image2` processing
+  * `tso-spec2`: Intended for `calwebb_tso_spec2` processing
```

Comment:
Shouldn't these now be "calwebb_tso-image2" and "calwebb_tso-spec2" (i.e. hyphens instead of underscore)?

Reply:
This was to have been merged before that issue came up. But good point, since this is still hanging around.

Reply:
Retry: This is part of #2557 and will be handed there.